rotkehlchen/chain/ethereum/modules/compound/utils.py

```diff
@@ -0,0 +1,17 @@
+from typing import TYPE_CHECKING, NamedTuple
+
+if TYPE_CHECKING:
+    from rotkehlchen.accounting.structures.balance import Balance, BalanceType
+    from rotkehlchen.fval import FVal
+
+
+class CompoundBalance(NamedTuple):
```

Comment:
Is this only for compound? This seems like a pretty generic structure. We don't use it anywhere else? Say Aave?

Reply:
This was only being used for compound. Aave has a different structure:
```py

class AaveLendingBalance(NamedTuple):
    """A balance for Aave lending.

    Asset not included here since it's the key in the map that leads to this structure
    """
    balance: Balance
    apy: FVal
    version: int

    def serialize(self) -> dict[str, str | dict[str, str]]:
        return {
            'balance': self.balance.serialize(),
            'apy': self.apy.to_percentage(precision=2),
        }


class AaveBorrowingBalance(NamedTuple):
    """A balance for Aave borrowing.

    Asset not included here since it's the key in the map that leads to this structure
    """
    balance: Balance
    variable_apr: FVal
    stable_apr: FVal
    version: int

    def serialize(self) -> dict[str, str | dict[str, str]]:
        return {
            'balance': self.balance.serialize(),
            'variable_apr': self.variable_apr.to_percentage(precision=2),
            'stable_apr': self.stable_apr.to_percentage(precision=2),
        }


class AaveBalances(NamedTuple):
    """The Aave balances per account"""
    lending: dict[CryptoAsset, AaveLendingBalance]
    borrowing: dict[CryptoAsset, AaveBorrowingBalance]
```